perllib/FixMyStreet/App/Controller/Open311.pm

```diff
@@ -233,44 +233,42 @@ sub output_requests : Private {
 
         my $request =
         {
-            'service_request_id' => [ $id ],
-            'title' => [ $problem->title ], # Not in Open311 v2
-            'detail'  => [ $problem->detail ], # Not in Open311 v2
-            'description' => [ $problem->title .': ' . $problem->detail ],
-            'lat' => [ $problem->latitude ],
-            'long' => [ $problem->longitude ],
-            'status' => [ $problem->state ],
-#            'status_notes' => [ {} ],
-            'requested_datetime' => [ w3date($problem->confirmed) ],
-            'updated_datetime' => [ w3date($problem->lastupdate) ],
-#            'expected_datetime' => [ {} ],
-#            'address' => [ {} ],
-#            'address_id' => [ {} ],
-            'service_code' => [ $problem->category ],
-            'service_name' => [ $problem->category ],
-#            'service_notice' => [ {} ],
-#            'zipcode' => [ {} ],
-            'interface_used' => [ $problem->service ], # Not in Open311 v2
```

Comment:
Yeah, bit brief! It's to stop the JSON output being `{ "service_request_id": [ 123456 ], "title": [ "TITLE" ], … }` which seemed a bit wrong! Adding the `NoAttr => 1` to the XML output stops that being any different (which is why I assume whoever wrote this had put these lists in in the first place).